charts/nfs-subdir-external-provisioner/templates/clusterrolebinding.yaml

```diff
@@ -14,3 +14,25 @@ roleRef:
   name: {{ template "nfs-subdir-external-provisioner.fullname" . }}-runner
   apiGroup: rbac.authorization.k8s.io
 {{- end }}
+
```

Comment:
i am wondering if it is not better to create file for this new object
anyway, the new lines here seems to be redundant